cmake/template/setup.bat.in

```diff
@@ -48,5 +48,5 @@ rem Set YARP and Gazebo related env variables
 set "YARP_DATA_DIRS=%YARP_DATA_DIRS%;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%\share\RRbot"
 set "GAZEBO_MODEL_PATH=%GAZEBO_MODEL_PATH%;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%\share\RRbot\robots"
 rem Configure the Matlab
-set "MATLABPATH=%MATLABPATH%;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%\mex;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%\mex\+wbc\simulink;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%\mex\+wbc\examples;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%\share\WBToolbox;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%\share\WBToolbox\images"
+set "MATLABPATH=%MATLABPATH%;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%\mex;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%\mex\+wbc\simulink;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%\mex\+wbc\simulink\MomentumVelocityControl;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%\mex\+wbc\examples;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%\share\WBToolbox;%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%\share\WBToolbox\images"
```

Comment:
Just a small comment:  I didn't know there were examples installed in  `%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX%\mex\+wbc\examples`, is whole-body-controller installing them or another repo? In case it is another repo, we'll have to pay attention when refactor the installation path in WBC v3.0